[2025-06-13T03:09:36.840+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-13T03:09:36.860+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: payparser_pipeline_dag.process_new_images manual__2025-06-13T03:09:34.826036+00:00 [queued]>
[2025-06-13T03:09:36.868+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: payparser_pipeline_dag.process_new_images manual__2025-06-13T03:09:34.826036+00:00 [queued]>
[2025-06-13T03:09:36.869+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-06-13T03:09:36.882+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_new_images> on 2025-06-13 03:09:34.826036+00:00
[2025-06-13T03:09:36.887+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=180) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-13T03:09:36.889+0000] {standard_task_runner.py:72} INFO - Started process 181 to run task
[2025-06-13T03:09:36.890+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'payparser_pipeline_dag', 'process_new_images', 'manual__2025-06-13T03:09:34.826036+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/payparser_dag.py', '--cfg-path', '/tmp/tmp19izkl4a']
[2025-06-13T03:09:36.893+0000] {standard_task_runner.py:105} INFO - Job 66: Subtask process_new_images
[2025-06-13T03:09:36.948+0000] {task_command.py:467} INFO - Running <TaskInstance: payparser_pipeline_dag.process_new_images manual__2025-06-13T03:09:34.826036+00:00 [running]> on host 9b9192f456bf
[2025-06-13T03:09:37.035+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zain' AIRFLOW_CTX_DAG_ID='payparser_pipeline_dag' AIRFLOW_CTX_TASK_ID='process_new_images' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T03:09:34.826036+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T03:09:34.826036+00:00'
[2025-06-13T03:09:37.036+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-13T03:09:37.060+0000] {logging_mixin.py:190} INFO - ⏳ Processing image: /opt/***/shared/downloads/WhatsApp Image 2025-06-10 at 01.06.41_99a98d8f.jpg
[2025-06-13T03:09:38.643+0000] {logging_mixin.py:190} INFO - Date formatting error: time data '10 Jun 2025 01:06 AM' does not match format '%d %B %Y %I:%M %p'
[2025-06-13T03:09:38.648+0000] {logging_mixin.py:190} INFO - Database error: unable to open database file
[2025-06-13T03:09:38.649+0000] {logging_mixin.py:190} INFO - ❌ Error processing WhatsApp Image 2025-06-10 at 01.06.41_99a98d8f.jpg: cannot access local variable 'conn' where it is not associated with a value
[2025-06-13T03:09:38.649+0000] {logging_mixin.py:190} INFO - ⏳ Processing image: /opt/***/shared/downloads/WhatsApp Image 2025-06-10 at 18.23.24_307f356c.jpg
[2025-06-13T03:09:40.039+0000] {logging_mixin.py:190} INFO - Date formatting error: time data '10 Jun 2025 06:23 PM' does not match format '%d %B %Y %I:%M %p'
[2025-06-13T03:09:40.040+0000] {logging_mixin.py:190} INFO - Database error: unable to open database file
[2025-06-13T03:09:40.041+0000] {logging_mixin.py:190} INFO - ❌ Error processing WhatsApp Image 2025-06-10 at 18.23.24_307f356c.jpg: cannot access local variable 'conn' where it is not associated with a value
[2025-06-13T03:09:40.045+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-13T03:09:40.055+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-13T03:09:40.055+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=payparser_pipeline_dag, task_id=process_new_images, run_id=manual__2025-06-13T03:09:34.826036+00:00, execution_date=20250613T030934, start_date=20250613T030936, end_date=20250613T030940
[2025-06-13T03:09:40.115+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-13T03:09:40.130+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-13T03:09:40.133+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
